src/routes/Manual.svelte

```diff
@@ -268,6 +269,7 @@ function handleActuatorClick(e) {
                 {/each}
             </ul>
         </div><div class='col-25'>
+            <br/><br/><br/>
```

Comment:
🤔

Reply:
That was just a temporary 'hacky' way to fix the spacing so the timing column lined up with messages at the top of the column on the left when I changed the layout so the hexagons don't get cutoff at the bottom. Will likely be replaced when I go through and finalize the layout for everything.